.azure-pipelines/generate-beta-rollup-module-template.yml

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file should be renamed to generate-rollup-module-template.yml. It generates both v1.0 and beta modules.